service/extractor.py

```
@@ -1,21 +1,496 @@
+        return {"readability_score": 0.0, "coherence_score": 0.0}
+    
+    # Базовые метрики
+    avg_words = sum(len([t for t in sent if not t.is_punct]) for sent in sentences) / len(sentences)
+    
+    return {
+        "readability_score": calculate_readability(doc),
+        "coherence_score": calculate_coherence(doc),
+        "avg_words_per_sentence": round(avg_words, 2)
+    }
+
+def create_fallback_structure(text: str) -> Dict[str, Any]:
```

Ruff (0.8.2)

Redefinition of unused `create_fallback_structure` from line 97

(F811)